web/skins/classic/views/js/event.js

```diff
@@ -690,8 +690,8 @@ function streamNext(action) {
   // How about we just reload the page?
```

Comment:
Why did you decide to do it this way?

Reply:
because I was lazy. Also much has changed since then.

Reply:
For now I'm transferring it to draft.
When I have time, I'll look at the code in more detail.

Reply:
I reviewed the code and added some comments, applied use strict etc.  

THe problem is that we might be coming from an event playing with zms to an event with video.js.  We need to be able to update the actual player in js.  At the time (the video.js stuff was new) it was easier to just reload. We didn't have fullscreen back then.  ANyways, it shouldn't be a big deal now.  

I want to transition the streaming code to an EventSTream.js module like MonitorStream.  So it can be reused for montage etc. I have a need for that in the short term, so I will be working on that.